src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1204,6 +1217,18 @@ namespace winrt::Microsoft::Terminal::Control::implementation
                                        const WORD scanCode,
                                        const ControlKeyStates modifiers,
                                        const bool keyDown)
+    {
+        // Broadcast the key  to all listeners
+        auto keySentArgs = winrt::make<KeySentEventArgs>(vkey, scanCode, modifiers, keyDown);
+        _KeySentHandlers(*this, keySentArgs);
+
+        return BroadcastKeyEvent(vkey, scanCode, modifiers, keyDown);
+    }
+
+    bool TermControl::BroadcastKeyEvent(const WORD vkey,

```

Comment:
This isn't exactly true, is it? It feels weird for the final port-of-call before we get out of TermControl to be called "Broadcast". To whom does it broadcast?